### PR TITLE
liquidsoap: Add version 1.4.3

### PR DESCRIPTION
**Pull request by earnestma**

### CODE REVIEW

File: bucket/liquidsoap.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "version": "1.4.3",
+    "description": "Audio and video streaming language",
+    "homepage": "https://www.liquidsoap.info",
+    "license": "GPL-2.0",
```
Comment (Ash258) [inline bucket/liquidsoap.json:5]: Invalid SPX identifier

File: bucket/liquidsoap.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+            "extract_dir": "liquidsoap-v1.4.3-win64"
+        }
+    },
+    "bin": "liquidsoap.exe",
+    "checkver": {
+        "github": "https://github.com/savonet/liquidsoap"
+    },
+    "autoupdate": {
+        "url": "https://github.com/savonet/liquidsoap/releases/download/v$version/liquidsoap-v$version-win64.zip",
+        "extract_dir": "liquidsoap-v$version-win64"
```
Comment (Ash258) [inline bucket/liquidsoap.json:19]: 64 bit

File: bucket/liquidsoap.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+    "version": "1.4.3",
+    "description": "Audio and video streaming language",
```
Comment (Ash258) [inline bucket/liquidsoap.json:3]: Not clear description

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/liquidsoap.json (hunk 1 above)
Comment (earnestma) [reply bucket/liquidsoap.json]: Their license didn't specify whether it was GPL-2.0-only or GPL-2.0-or-later, so it's now GPL-2.0-only unless they change/ specify it.

Comment (earnestma) [reply bucket/liquidsoap.json]: Nope. On second thought it makes more sense to include later versions.

> This program is free software; you can redistribute it and/or modify
    it under the terms of the GNU General Public License as published by
    the Free Software Foundation; either version 2 of the License, or
    (at your option) any later version.

**Review by Ash258 — APPROVED** [review]

**Pull request merged**